envoy-control-runner/src/main/kotlin/pl/allegro/tech/servicemesh/envoycontrol/snapshot/debug/SnapshotDebugController.kt

```diff
@@ -90,6 +94,50 @@ class SnapshotDebugController(controlPlane: ControlPlane) {
         }
     }
 
+    @GetMapping("/snapshot-global/{service}")
+    fun serviceSnapshot(
+        @PathVariable service: String,
+        @RequestParam dc: String?,
+        @RequestParam(defaultValue = "false") xds: Boolean
+    ): ResponseEntity<EndpointInfoList> {
+        val updateResult = snapshotUpdater.getGlobalSnapshot()
```

Comment:
controller nie powinien mieć tak skomplikowanej logiki a bardziej wskazywać na serwis w którym to sie wykona